src/frontend/web_application/src/components/DeviceForm/presenter.jsx

```diff
@@ -0,0 +1,145 @@
+import PiBar from '../../components/PiBar/presenter';
+import { FormGrid, FormRow, FormColumn, Fieldset, Legend, TextFieldGroup, SelectFieldGroup } from '../../components/form';
+
+import './style.scss';
+
+const DeviceForm = ({ device, __ }) => {
+  const thisDevice = device;
+  const thisDeviceIPs = Array.from(new Set(thisDevice.ips));
+  const lastSeenDate = new Date(thisDevice.last_seen).toLocaleDateString();
+  const insertDate = new Date(thisDevice.date_insert).toLocaleDateString();
+  const deviceTypes = [
+    { value: 'desktop', label: __('device.type.desktop') },
+    { value: 'laptop', label: __('device.type.laptop') },
+    { value: 'smartphone', label: __('device.type.smartphone') },
+    { value: 'tablet', label: __('device.type.tablet') },
+  ];
+
+  return (
+    <div className="m-device">
+
+      <div className="m-device__section m-device__pi">
+        <PiBar level={thisDevice.pi ? thisDevice.pi : 0} />
+      </div>
+
+      <FormGrid className="m-device__section m-device__form">
+        <Title align="center" className="m-device__title">{__('device.manage.title')}</Title>
+        <FormRow className="m-device__intro"><p>{__('device.manage.descr')}</p></FormRow>
+        <Fieldset>
+          <Legend>{__('device.manage_form.name.label')}</Legend>
+          <FormRow reverse>
+            <FormColumn bottomSpace size="medium" className="m-device__infotext">
+              <label htmlFor="device-name">{__('device.manage_form.name.infotext')}</label>
+            </FormColumn>
+            <FormColumn bottomSpace size="medium">
+              <TextFieldGroup
+                label={__('device.manage_form.name.label')}
+                name="device-name"
+                id="device-name"
+                showLabelforSr
+                key={thisDevice.name}
+                defaultValue={thisDevice.name}
+              />
+            </FormColumn>
+          </FormRow>
+        </Fieldset>
+        <Fieldset>
+          <Legend>{__('device.manage_form.type.label')}</Legend>
+          <FormRow reverse>
+            <FormColumn bottomSpace size="medium" className="m-device__infotext">
+              <label htmlFor="device-type">{__('device.manage_form.type.infotext')}</label>
+            </FormColumn>
+            <FormColumn size="medium">
+              <SelectFieldGroup
+                className="m-device__type"
+                label={__('device.manage_form.type.label')}
+                name="device-type"
+                id="device-type"
+                showLabelforSr
+                value={thisDevice.type}
+                options={deviceTypes}
+              />
+            </FormColumn>
+          </FormRow>
+        </Fieldset>
+        <Fieldset>
+          <Legend>{__('device.manage_form.ips.label')}</Legend>
+          <FormRow reverse>
+            <FormColumn bottomSpace className="m-device__infotext" size="medium">
+              <label htmlFor="device-ips">{__('device.manage_form.ips.infotext')}</label>
```

Comment:
I'm not sure this is useful to set as label just informative text. so reverse modifier on FormRow might not be useful